src/encoding/toHex.ts

```diff
@@ -45,7 +45,10 @@ const numberToHex = (num: number | bigint): Hex => {
   const maxValue = BigInt(Number.MAX_SAFE_INTEGER);
   const minValue = 0;
 
-  if ((maxValue && value > maxValue) || value < minValue) {
+  if (
```

Comment:
Better remove bigint type from possible inputs then.

Reply:
why is it?

Reply:
Nevermind. There is `numberToHex` function. Better remove `bigint` from its parameters and add new function `bigintToHex`